server/src/main/java/org/apache/druid/segment/realtime/RealtimeMetricsMonitor.java

```diff
@@ -137,6 +114,12 @@ public boolean doMonitor(ServiceEmitter emitter)
       emitter.emit(builder.build("ingest/handoff/count", metrics.handOffCount() - previous.handOffCount()));
       emitter.emit(builder.build("ingest/sink/count", metrics.sinkCount()));
       emitter.emit(builder.build("ingest/events/messageGap", metrics.messageGap()));
+
+      long maxSegmentHandoffTime = metrics.maxSegmentHandoffTime();
+      if (maxSegmentHandoffTime >= 0) {
+        emitter.emit(builder.build("ingest/handoff/time", maxSegmentHandoffTime));
```

Comment:
I wonder if we should just keep the default as 0 rather than -1 and always emit the `maxSegmentHandoffTime`, even if it is 0. This would also match the behaviour of the other metrics being emitted here, especially `handoff/count`.

Reply:
I think this metric isn't of much value when segments aren't being handed off and emitting 0 wouldn't be very helpful.